src/Idris/Package/Init.idr

```diff
@@ -67,25 +68,50 @@ prompt p = putStr p >> fflush stdout >> getLine
 
 export
 covering
-interactive : IO PkgDesc
+interactive : IO (Either () PkgDesc)
```

Comment:
Using Either () instead of Maybe is a bit unconventional, is there a specific reason for that?